uqcsbot/utils/snailrace_utils.py

```diff
@@ -0,0 +1,146 @@
+import discord, random, datetime, asyncio
+
+# Racer Icon
+SNAILRACE_SNAIL_EMOJI = "🐌"
+
+# Entry opening time
+SNAILRACE_OPEN_TIME = 10
+SNAILRACE_STEP_TIME = 1
+
+# The number of characters in the track
+SNAILRACE_TRACK_LENGTH_CHARS = 15
+
+# The length of the track in steps
+SNAILRACE_TRACK_LENGTH = 100
```

Comment:
General comment/question:

We have a track of 100 steps, and 15 characters - so 6.66 steps per character? The race happens at a good pace, but fractional characters aren't ideal. Is it worth extending the output to 20 characters, just so we have even numbers?

(Also, I might be misunderstanding the logic here, so correct me if I'm wrong.)

Reply:
I have it as a fractional character on purpose to bring tension for some random moves in the race. It was a quirk of the original "snail race" thing I used to play with back in 2014. So I probably wont change is as it adds character to the race. 

Unless you really want me to change it.

Reply:
That's totally fair, carry on!